docs/about/index.rst

```diff
@@ -17,13 +42,118 @@ Code of conduct
+fundamental fairness issues before applying specific algorithms, and that
+mitigating unfairness in binary classification models is a relatively rare use
+case.
+They also discovered that fairness assessment is a common need, along with
+access to domain-specific guides to fairness metrics and unfairness mitigation
+algorithms.
+Additionally, many use cases take the form of regression or ranking, rather
+than classification.
+As a result of these insights, fairness assessment and use-case notebooks
+became key components of Fairlearn.
+Fairlearn also focuses on machine learning tasks beyond binary classification.
+
+The project was greatly expanded in the second half of 2019 thanks to the
+involvement of many contributors from Azure ML and Microsoft Research.
+At that time, the project started to have regular releases.
+
+.. _roadmap:
+
+Project focus areas
+-------------------
+
+As an open-source project, Fairlearn strives to incorporate the best of
+research and practice.
+AI is a rapidly evolving field, and fairness in AI is all the more so.
+We therefore encourage researchers, practitioners, and other stakeholders to
+contribute fairness metrics and assessment tools, unfairness mitigation algorithms,
+case studies and other educational materials to Fairlearn as we experiment,
+learn, and evolve the project together.
+
+Below we list the key areas that we prioritize in the short
+and medium term, but we are happy to consider other directions
+if they are aligned with the mission of Fairlearn and there is enough commitment
+from the contributors. If you want to get involved, please
+:ref:`reach out <communication>`. For concrete opportunities and
+work in progress please review our `project boards <https://github.com/fairlearn/fairlearn/projects>`_.
+
+Short and medium-term focus areas
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*Last update: August 12, 2020*
+
+#. *Decrease adoption barriers for the current assessment and mitigation tools in Fairlearn*
+
+   - **Improve existing notebooks and create new notebooks** that
+     `make fairness issues concrete https://fairlearn.github.io/contributor_guide/contributing_example_notebooks.html>`_
+     while demonstrating how to apply Fairlearn. To accelerate this process, we are experimenting with
+     :ref:`weekly sessions <notebook_deep_dive>` where people can discuss
+     individual example notebooks in detail.
+     
+   - **Improve the project documentation**: critique the current content, expand user guides,
+     improve writing and examples, port the current Python documentation to the numpydoc format. 
+     Convert existing example notebooks from `.ipynb` into `.py` files that
+     `render properly on the website <https://fairlearn.github.io/auto_examples/notebooks/index.html>`_,
+     while still being downloadable as `.py` or `.ipynb` files.
+
+   - **Improve the usability, relevance, and look of Fairlearn website** with the audience of practitioners in mind.
```

Comment:
This link goes to the initial proposal I wrote about the webpage. Perhaps some things will change about this? I'm not sure indicating that that proposal is what we're aiming towards is right. To me this should rather be something along the lines of "we're considering ways of making the website relevant and engaging for practitioners" and we can link to a new proposal once that's ready.

Reply:
I've created a new project board and linked to it.